src/features/Application/validationSchema.ts

```diff
@@ -17,8 +17,8 @@ const getValidationSchema = (isCreate: boolean, pageNumber: number) => {
               fieldOfStudy: string().required('Required'),
               graduationYear: number()
                 .required('Required')
-                .min(2019)
-                .max(2025),
+                .min(2020, 'Graduation year must be 2020 or later')
```

Comment:
MLH Guidelines allow for students that have graduated within the last 12 months to attend league events. That's why last year was 2018 and this year should be 2019.

Reply:
Also, while we're adding constants (or env variables 😉) I think these years should also be added to that file. 

The theory being that the only thing that needs to change from hackathon to hackathon is this one env/constants file.